### PR TITLE
Designs header and header navigation

### PR DESCRIPTION
**Pull request by mikaelbr**
Adds header with links to central places (not on the same page). Also added some effects to make the site more "dynamic". See video and image

https://www.dropbox.com/s/6d5gqm2k27s79oo/demo.mov?dl=0

image

### CODE REVIEW

File: src/index/index.tsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
 import React from "react";
 import Layout from "src/layout";
-
+import { useReducedMotionEffect } from "src/utils/hooks";
 import style from "./index.module.css";
 
 const Home = () => {
+  const reducedMotion = useReducedMotionEffect();
+  console.log("reduced", reducedMotion);
```
Comment (adriansberg) [inline src/index/index.tsx:8]: Remove

File: src/layout/index.tsx [hunk 2]
```diff
@@ -23,7 +24,38 @@ const Layout: React.FC<LayoutProps> = ({
...
+            <Link href="/">
+              <a>
+                <img src={require("./variant.svg")} alt="Variant" />
+              </a>
+            </Link>
+          </h1>
+          <nav className={style.header__nav}>
+            <ul>
+              <li>
+                <a href="https://jobs.variant.no" rel="noopner">
```
Comment (adriansberg) [inline src/layout/index.tsx:39]:
```suggestion
                <a href="https://jobs.variant.no" rel="noopener">
```

File: src/layout/index.tsx [hunk 3]
```diff
@@ -23,7 +24,38 @@ const Layout: React.FC<LayoutProps> = ({
...
+                  Bli en variant
+                </a>
+              </li>
+              <li>
+                <a href="http://variant.blog" rel="noopner">
```
Comment (adriansberg) [inline src/layout/index.tsx:44]:
```suggestion
                <a href="http://variant.blog" rel="noopener">
```

File: src/layout/index.tsx [hunk 4]
```diff
@@ -23,7 +24,38 @@ const Layout: React.FC<LayoutProps> = ({
...
+                  Blogg
+                </a>
+              </li>
+              <li>
+                <a href="http://handbook.variant.no" rel="noopner">
```
Comment (adriansberg) [inline src/layout/index.tsx:49]:
```suggestion
                <a href="http://handbook.variant.no" rel="noopener">
```

File: src/index/index.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/index/index.tsx]: I thought I pushed that amending... 😳

Comment (adriansberg) [reply src/index/index.tsx]: Looks like you did, "Outdated"

Comment (mikaelbr) [reply src/index/index.tsx]: Peewh

**Pull request merged**